docs/src/modules/components/AppDrawer.js

```diff
@@ -163,7 +163,7 @@ function AppDrawer(props) {
       </div>
       <Divider />
       <Box mx={3} my={2}>
-        <DiamondSponsors />
+        {/* <DiamondSponsors spot="sidenav" /> */}
```

Comment:
Should this be removed or was this just commented out for debugging?

Reply:
I'm interested in seeing the volume of click with a single placement. Also waiting for feedback.

Reply:
> Also waiting for feedback.

About its removal in the drawer? Should you maybe include this request for review in the PR description?

Reply:
waiting for feedback, I meant from sencha.

Reply:
I assume this feedback has been received? If so and the new location is good we should probably remove the commented code for cleanliness. If not I would still remove the code but why are we merging without feedback?

Reply:
I have removed the comment